src/gtl/components/data-table/data-table.html

```diff
@@ -23,8 +23,14 @@
         </div>
       </div>
     </div>
+    <div class="col-md-1 col-lg-1 col-xs-1 miq-page-counter">
+      <label>{{tableCtrl.settings.startIndex + 1}} - {{tableCtrl.settings.endIndex + 1}}
+        {{tableCtrl.MiQTranslateService.translateString('of')}}
```

Comment:
I think for i18n, this would have to be more like "%d - %d of %d" .. otherwise, there's too many ways of translating "of" that don't make sense.